tests/test-war-smoke/pom.xml

```diff
@@ -152,12 +121,13 @@
     </profile>
     <profile>
       <id>test.remote</id>
-      <activation>
-        <property>
-          <name>test.mode</name>
-          <value>remote</value>
-        </property>
-      </activation>
+      <properties>
+        <maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
+        <gcloud-projectId-file>${project.build.directory}/gcloud-projectid.properties
+        </gcloud-projectId-file>
+        <!-- remote tests require image in gcr.io -->
+        <jetty.test.image>gcr.io/${app.deploy.project}/jetty:${docker.tag.short}</jetty.test.image>

```

Comment:
@meltsufin do you know there are two different usages of maven.build.timestamp.format?  the TLP has it as yyyy-MM-dd_HH_mm but the one that passes it into the appengine deploy in tests is yyyyMMdd-HHmm